client/app/core/constants.js

```diff
@@ -4,6 +4,8 @@
 
   angular.module('app.core')
     .constant('lodash', _)
+    // eslint-disable-next-line
```

Comment:
This one can't be fixed. It's a global, which is not defined in this file :) so.. we really have to disable the rule here (or do something ugly like `window["ActionCable"]`)